mdnDocsOverrides.json

```
@@ -350,19 +370,28 @@
   "capture-stream-audio-video": [
     "Web/API/HTMLMediaElement/captureStream"
   ],
-  "capture-stream-canvas": [],
+  "capture-stream-canvas": [
+    "Web/API/CanvasCaptureMediaStreamTrack"
+  ],
   "caret-color": [
     "Web/CSS/caret-color"
   ],
   "cascade-layers": [
     "Web/CSS/@layer"
   ],
-  "case-insensitive-attributes": [],
-  "case-sensitive-attributes": [],
+  "case-insensitive-attributes": [
+    "Web/CSS/Attribute_selectors#s"
+  ],
+  "case-sensitive-attributes": [
+    "Web/CSS/Attribute_selectors#s"
+  ],
   "ch": [
     "Web/CSS/length#ch"
   ],
-  "channel-messaging": [],
+  "channel-messaging": [
+    "Web/API/MessageChannel",
+    "Web/API/MessagePort"
+  ],
```

This feature is all about the Channel Messaging API, which happens to have a nice API page on MDN: Web/API/Channel_Messaging_API, so I think we might as well just link to this to let people learn.

```suggestion
  "channel-messaging": [
    "Web/API/Channel_Messaging_API"
  ],
```